test/integration/short/timeout-tests.js

```diff
@@ -7,6 +7,7 @@ var Connection = require('../../../lib/connection');
 var utils = require('../../../lib/utils');
 var types = require('../../../lib/types');
 var errors = require('../../../lib/errors');
+var ExecutionProfile = require('../../../lib/execution-profile').ExecutionProfile;
 
 describe('client read timeouts', function () {
   this.timeout(120000);
```

Comment:
Pushed that change.  It shortens timeout-tests from ~7-8 mins to 2 mins, although somewhat curious to see how reliably it runs in CI as i'm worried there might be a timing window after resuming a node where it isn't responsive, but usually unpausing gets thing in order really quick.  We'll see how it goes.